ibu-imager/postpivot/postpivot.go

```diff
@@ -338,7 +341,7 @@ func (p *PostPivot) cleanup() error {
 	return utils.RemoveListOfFolders(p.log, []string{p.workingDir, common.SeedDataDir})
 }
 
-func (p *PostPivot) setNewClusterID(ctx context.Context, client runtimeclient.Client, clusterInfo *clusterinfo.ClusterInfo) error {
+func (p *PostPivot) setNewClusterID(ctx context.Context, client runtimeclient.Client, clusterInfo *clusterconfig_api.SeedReconfiguration) error {
```

Comment:
What will happen if the clusterInfo.ClusterID is empty?

Reply:
For now, an error. `// TODO: Generate a fresh cluster ID in case it is empty`